docs/swagger/spec.yml

```diff
@@ -591,6 +591,41 @@ paths:
           description: The UUID parameter does not refer to a project or the user does not have access to the project it refers to
           schema:
             $ref: '#/definitions/Error'
+
+  /projects/{uuid}/aoi/:
+    get:
+      summary: Get all Areas of Interest for this project.
+      parameters:
+        - $ref: '#/parameters/uuid'
+      responses:
+        200:
+          description: Json summary of the filters and Polygons that make up AOIs for the given project.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/AOI'
```

Comment:
How come? Do we expect many AOIs per single Project? Note that I admit total ignorance over what Pagination is used for in RF and how it works.